discovery/pkg/sync/queue.go

```diff
@@ -52,17 +54,20 @@ func NewQueue(logger *logrus.Logger, clusterName string, kubeClient kubernetes.I
 		Threadiness: threadiness,
 		Metrics:     metrics,
 		ClusterName: clusterName,
+		ClusterType: clusterType,
 	}
 }
 
 // Action that is added to the queue for processing
 type Action interface {
```

Comment:
I am wondering if this should be called "Resource" or something... Not this PRs concern though